bodhi/server/scripts/approve_testing.py

```diff
@@ -35,7 +35,9 @@
 
 
 logger = logging.getLogger('approve-testing')
-logging.basicConfig(level=logging.ERROR)
+if __name__ == '__main__':
```

Comment:
That won't work, when the script is run as `bodhi-approve-testing` its `__name__` value is `bodhi.server.scripts.approve_testing`.
You could, for example, check that `logger.handlers` is an empty list or not. If it's empty, configure logging, otherwise do nothing.
This also looks like something that should be done in `main()`, not in the module itself.

Reply:
Thanks, I'll change it.